photon-client/src/views/PipelineViews/PnPTab.vue

```diff
@@ -60,7 +60,7 @@
         },
         data() {
             return {
-                targetList: ['2020 High Goal Outer', '2020 High Goal Inner', '2019 Dual Target', 'Power Cell (7in)', '2016 High Goal'], //Keep in sync with TargetModel.java
+                targetList: ['2020 High Goal Outer', '2020 High Goal Inner', '2019 Dual Target', 'Power Cell (7in)','2022 Cargo Ball (9.5)', '2016 High Goal'], //Keep in sync with TargetModel.java
```

Comment:
Nitpick, but could the naming be more consistent here? Maybe "2020 Powercell (7in)" and "2022 Cargo Ball (9.5in)"?

Reply:
Yeah I can. I'll try to get that done soon.